backend/src/controllers/CommentController.ts

```diff
@@ -0,0 +1,28 @@
+import {Request, Response} from "express";
+import CommentSchema from "../models/CommentModel";
+import PostSchema from "../models/PostModel";
+
+
+export const createComment  = async (req: Request, res: Response) => {
+    try{
+

```

Comment:
it would be cool to add formatting here as well, I can see that this file is not formatted properly (missing whitespaces, empty lines where they shouldn't be empty)

Reply:
fixed